omegaconf/_utils.py

```diff
@@ -192,6 +192,12 @@ def _resolve_forward(type_: Type[Any], module: str) -> Type[Any]:
         return type_
 
 
+def get_attr_class_field_names(obj: Any) -> List[str]:
+    is_type = isinstance(obj, type)
+    obj_type = obj if is_type else type(obj)
+    return [name for name in attr.fields_dict(obj_type).keys()]
```

Comment:
Minor simplification:
```suggestion
    return list(attr.fields_dict(obj_type))
```